diff --git a/src/plugins/crypto_openssl/CMakeLists.txt b/src/plugins/crypto_openssl/CMakeLists.txt
deleted file mode 100644
index d014144ec..000000000
--- a/src/plugins/crypto_openssl/CMakeLists.txt
+++ /dev/null
@@ -1,26 +0,0 @@
-# Copyright (c) 2018 Cisco and/or its affiliates.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at:
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-if(NOT OPENSSL_FOUND)
-  return()
-endif()
-
-include_directories(${OPENSSL_INCLUDE_DIR})
-
-add_vpp_plugin(crypto_openssl
-  SOURCES
-  main.c
-
-  LINK_LIBRARIES
-  ${OPENSSL_LIBRARIES}
-)
diff --git a/src/plugins/ikev2/CMakeLists.txt b/src/plugins/ikev2/CMakeLists.txt
deleted file mode 100644
index dac246524..000000000
--- a/src/plugins/ikev2/CMakeLists.txt
+++ /dev/null
@@ -1,38 +0,0 @@
-# Copyright (c) 2018 Cisco and/or its affiliates.
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at:
-#
-#     http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-add_definitions (-DWITH_LIBSSL=1)
-include_directories(${OPENSSL_INCLUDE_DIR})
-
-add_vpp_plugin(ikev2
-  SOURCES
-  ikev2.c
-  ikev2_api.c
-  ikev2_cli.c
-  ikev2_crypto.c
-  ikev2_format.c
-  ikev2_payload.c
-
-  API_FILES
-  ikev2.api
-
-  API_TEST_SOURCES
-  ikev2_test.c
-
-  INSTALL_HEADERS
-  ikev2.h
-  ikev2_priv.h
-
-  LINK_LIBRARIES
-  ${OPENSSL_LIBRARIES}
-)
diff --git a/src/plugins/tlsopenssl/tls_openssl.c b/src/plugins/tlsopenssl/tls_openssl.c
index 6d0364c3a..652c52466 100644
--- a/src/plugins/tlsopenssl/tls_openssl.c
+++ b/src/plugins/tlsopenssl/tls_openssl.c
@@ -553,7 +553,7 @@ openssl_ctx_init_client (tls_ctx_t * ctx)
       return -1;
     }
 
-  SSL_CTX_set_ecdh_auto (oc->ssl_ctx, 1);
+  //SSL_CTX_set_ecdh_auto (oc->ssl_ctx, 1);
   SSL_CTX_set_mode (oc->ssl_ctx, SSL_MODE_ENABLE_PARTIAL_WRITE);
 #ifdef HAVE_OPENSSL_ASYNC
   if (om->async)
@@ -669,7 +669,7 @@ openssl_start_listen (tls_ctx_t * lctx)
     }
 #endif
   SSL_CTX_set_options (ssl_ctx, flags);
-  SSL_CTX_set_ecdh_auto (ssl_ctx, 1);
+  //SSL_CTX_set_ecdh_auto (ssl_ctx, 1);
 
   rv = SSL_CTX_set_cipher_list (ssl_ctx, (const char *) om->ciphers);
   if (rv != 1)
