From 47904f922ac042c24d3c8d4ecfda15d902c48c2e Mon Sep 17 00:00:00 2001
From: "xiaolongx.jiang" <xiaolongx.jiang@intel.com>
Date: Fri, 8 May 2020 07:06:30 +0000
Subject: [PATCH] openssl3.0.0-alpha1

Signed-off-by: xiaolongx.jiang <xiaolongx.jiang@intel.com>
---
 VERSION | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/VERSION b/VERSION
index b7529b4..bfcd4fc 100644
--- a/VERSION
+++ b/VERSION
@@ -3,5 +3,5 @@ MINOR=0
 PATCH=0
 PRE_RELEASE_TAG=alpha1
 BUILD_METADATA=
-RELEASE_DATE="23 Apr 2020"
+RELEASE_DATE=23 Apr 2020
 SHLIB_VERSION=3
-- 
2.17.1

